### PR TITLE
src/en/news/blog/2022/ceph-use-case-survey-2022: write blog about user survey

### PR DESCRIPTION
**Pull request by ljflores**
Signed-off-by: Laura Flores <lflores@redhat.com>

### CODE REVIEW

**Comment by ljflores** [comment]
https://wip-user-survey.ceph.io/en/news/blog/2022/ceph-use-case-survey-2022/

**Comment by ljflores** [comment]
@zdover23 let me know what you think. I tried to preserve your "commercial break" structure while also synthesizing each clump of use cases under a common theme. 

Please let me know if any of the grammar / sentence structure is off! And I think I will reformat the pie charts tomorrow so they don't leave a huge space.

**Comment by zdover23** [comment]
> @zdover23 let me know what you think. I tried to preserve your "commercial break" structure while also synthesizing each chunk of use cases under a common theme.
> 
> Please let me know if any of the grammar / sentence structure is off! And I think I will reformat the pie charts tomorrow so they don't leave a huge space.
> 
> You can see the blog post rendered here: https://wip-user-survey.ceph.io/en/news/blog/2022/ceph-use-case-survey-2022/

Laura, I have made about 50 comments, but they are all listed as "Pending". Can you see them?

**Review by zdover23 — CHANGES_REQUESTED** [review]

**Review by zdover23 — APPROVED** [review]

**Comment by ljflores** [comment]
@zdover23 I can see them!

**Comment by zdover23** [comment]
> @zdover23 I can see them!

Qapla!

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:16]: Fixed

Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:16]: Fixed.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:18]: Fixed

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:20]: Fixed

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:22]: Fixed

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:18]: Fixed.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:20]: Fixed

Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:20]: Fixed

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:79]: See other comment; it's to distinguish the survey population. We can remove Ceph user though, and just distinguish if someone is additionally a developer, open source contributor, etc.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (zdover23) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:73]: Fair enough. As I read on, it dawned on me that this was the distinction you meant to draw. I respect it.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:80]: How about "For development in the Ceph project"?

**Comment by zdover23** [comment]
> > @zdover23 let me know what you think. I tried to preserve your "commercial break" structure while also synthesizing each chunk of use cases under a common theme.
> > Please let me know if any of the grammar / sentence structure is off! And I think I will reformat the pie charts tomorrow so they don't leave a huge space.
> > You can see the blog post rendered here: https://wip-user-survey.ceph.io/en/news/blog/2022/ceph-use-case-survey-2022/
> 
> Laura, I have made about 50 comments, but they are all listed as "Pending". Can you see them?

Update, thirteen hours later: I had to "start a review" in order for my pending comments to appear. That's GitHub shrug emoji.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:82]: It means he develops features for the Ceph dashboard component. I can specify this if it makes it clearer.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:91]: This is a direct quote from Enrico. I get how some parts of it can be confusing though. I'll try paraphrasing it and removing the quote block so it's clear that we've paraphrased instead of direct-quoted.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (zdover23) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:82]: I think that would be helpful. Your prepositional phrase is not ambiguous, which is impressive in a for-phrase deployed in a technical sentence. His in-phrase nested in a for-phrase is a classic example of forcing "for" to do more work than it is capable of even at its strongest.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:94]: Direct quote from Enrico! XD

I've paraphrased it now so it's more readable. I am personally okay with this section being a bit more complex since 1) there's really no easy way to simplify this further without losing sight of Enrico's original comment, and 2) this part might appeal to more advanced Ceph users. Although the primary target audience is Grace Hopper, it's also for the Ceph community in general, which includes advanced users who would want to know this use case.

That said, let me know if there's anything I can to do to further make this simpler.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (zdover23) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:91]: Cool. Don't wait on me to get back to you with a rubber stamp of approval. I have complete confidence in your judgment, and you already have the bureaucratic GitHub PR approval that you need from me to merge this PR, so don't feel that I have to be consulted on this matter. When it looks right to you, it's right.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:110]: Also should say RGW pools!

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (zdover23) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md:80]: I think "for development in the Ceph project" still sounds pretty vague, like he might be doing primal scream therapy and staring at the ocean like Don Draper doing a guided meditation... only he's doing that stuff (personal development) somehow in the context of the Ceph project.

After checking out Junior's presence in the git log, I suggest this:

> Kamoltat Sirivadhna, a Ceph developer, uses Ceph to develop features for the Ceph project.

**Comment by ljflores** [comment]
@zdover23 I fixed stuff on one pass through; I plan to take another whack at this later today.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (zdover23) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md]: As I said somewhere else in this epistolary novel we're writing: when it looks good to you, it's ready for prime time.

**Comment by zdover23** [comment]
> @zdover23 I fixed stuff on one pass through; I plan to take another whack at this later today.

@ljflores Merge at will. You have the conn.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md]: Fixed.

File: src/en/news/blog/2022/ceph-use-case-survey-2022/index.md
Comment (ljflores) [reply src/en/news/blog/2022/ceph-use-case-survey-2022/index.md]: Fixed.

**Comment by ljflores** [comment]
Merged! Thanks @zdover23!

**Pull request merged**
